autoscheduler/frontend/src/hooks/useThunkDispatch.ts

```diff
@@ -0,0 +1,8 @@
+import { useDispatch } from 'react-redux';
+import { AnyAction } from 'redux';
+import { ThunkDispatch } from 'redux-thunk';
+import { RootState } from '../redux/reducer';
+
+export default function useThunkDispatch(): ThunkDispatch<RootState, undefined, AnyAction> {
```

Comment:
This literally just typecasts `useDispatch` so that typescript doesn't get upset at it, right? Would appreciate a comment as to why/when this is necessary